### ISSUE TITLE
(WCAG 2.1) Fix heading level order on application preview and within application pages

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3554]
Required to meet WCAG 2.1 guidelines. Identified during an accessibility audit against WCAG 2.1.

**WCAG SC**:  [1.3.1. Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)

**Observed results**:
- On the application preview page, there is a h2 preceding an h1. Headings should be in order. (screenshot 1 below)
- In the application screens, (aka when within a form), there is a hidden h1 heading and then a visual h2 heading. (Tracked separately in https://github.com/civiform/civiform/issues/2640). (screenshot 2 below)

**How to fix**: "Ensure heading level matches the heading's visual importance/level. Start with `<h1>` (typically just one) followed by `<h2>`, and so on." - a11y team

My suggestion is to:
- On preview page, switch the h1 and h2 labeling. So, keep everything the same visually, but mark "Accessibility primary" as the h1, and "Program application preview" as the h2. I believe the code for the "Accessibility primary" text is shared across the preview page and within the form, so this should address both issues.

![image](https://user-images.githubusercontent.com/9094240/192902915-39b38179-f900-4285-b400-3fa7e515e9a4.png)

![image](https://user-images.githubusercontent.com/9094240/192913212-53fc9172-71a1-4492-9076-a7138ae7f4b4.png)
